ably/doc.go

```diff
@@ -24,9 +24,12 @@
 // The OffAll method is like Off, except it is like calling all the "off"
 // functions.
 //
-// Calls to handlers for a single or for subsequent events may be concurrent,
-// but are always ordered. That is, if an event is fired after another, the same
-// handler will be called in the same order that the events were fired.
+// Each handler is assigned its own sequential queue of events. That is, any
```

Comment:
Does `handlers emit events here sequentially (FIFO)` ? If it does, it should be mentioned explicitly.

Reply:
It says: "you can count on events or messages to be delivered to the handler in the same order they were emitted". That's pretty explicit, isn't it?

Reply:
Mentioning, queue or just FIFO makes things explicit at a good level. That said, I will keep that change to be optional. But, I strongly feel it will help emitter behaviour more better.